components/kyma-environment-broker/internal/broker/services.go

```diff
@@ -33,8 +35,10 @@ func NewServices(cfg Config, log logrus.FieldLogger) *ServicesEndpoint {
 //   GET /v2/catalog
 func (b *ServicesEndpoint) Services(ctx context.Context) ([]domain.Service, error) {
 	var availableServicePlans []domain.ServicePlan
+	// we scope to the kymaruntime service only
+	class := b.servicesConfig[KymaServiceName]
```

Comment:
if the key `kymaruntime` does not exist in catalog.yaml (or has e.g. type) you will not get any error message (errror log) and plans will not contain description/metadata info.